### PR TITLE
Add logger service

### PR DESCRIPTION
**Pull request by ciremusyoka**
Introduces use of [morgan](https://www.npmjs.com/package/morgan) and [winston](https://www.npmjs.com/package/winston) logging packages.
- morgan is used as middleware for monitor requests
- winston is used to persist the the requests captured by morgan to a file

Addresses #12

### CODE REVIEW

File: src/configs/winston.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import * as winston from 'winston';
+import os from 'os';
+
+const homedir = os.homedir();
+const logsPath = `${homedir}/.express/logs/reveal-express-server.log`
```
Comment (moshthepitt) [inline src/configs/winston.ts:5]: Can we make this configurable please

File: src/configs/winston.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    maxsize: 5242880, // 5MB
+    maxFiles: 5,
+    colorize: false,
+    format: winston.format.simple(),
+  },
+  console: {
+    level: 'debug',
+    handleExceptions: true,
+    json: false,
+    colorize: true,
```
Comment (moshthepitt) [inline src/configs/winston.ts:23]: Do we need to make any of these configurable?

**Review by moshthepitt — CHANGES_REQUESTED** [review]

File: src/configs/winston.ts (hunk 1 above)
Comment (ciremusyoka) [reply src/configs/winston.ts:5]: Sure.
Was looking at the possibility of using the default logs folder `/var/log/` but am experiencing some problems with write permissions.

File: src/configs/winston.ts (hunk 2 above)
Comment (ciremusyoka) [reply src/configs/winston.ts]: addressed

File: src/app/tests/index.loging.test.ts [hunk 3]
```diff
@@ -0,0 +1,139 @@
+/* eslint-disable @typescript-eslint/camelcase */
```
Comment (moshthepitt) [inline src/app/tests/index.loging.test.ts:1]: There's a typo in `index.loging.test.ts` (logging)

**Review by moshthepitt — APPROVED** [review]

**Review by KipSigei — APPROVED** [review]

**Pull request merged**